src/System.Private.ServiceModel/tools/test/Bridge/Bridge/Program.cs

```diff
@@ -18,6 +19,7 @@ internal class Program
         internal const bool DefaultAllowRemote = false;
         internal const string DefaultRemoteAddresses = "LocalSubnet";
         internal const string BridgeControllerEndpoint = "Bridge";
+        internal static readonly TimeSpan DefaultRequireBridgeTimeSpan = TimeSpan.FromMinutes(5);
```

Comment:
Do we really want to default to taking 5 minutes to decide we are unable to start the bridge?